### ISSUE TITLE
Accessibility: On Library Videos page, the video iframe doesn't have a title attribute

### ISSUE THREAD

**Issue by sandbergja** [subjectsplus/SubjectsPlus#1368]
Steps to reproduce:
1. Go to [your subjects plus URL]/subjects/video.php
2. Click on a video
3. Note that the iframe containing the YouTube video doesn't have the required title attribute.

This is mentioned on this blog post (along with the drupal and wordpress solutions): http://adrianroselli.com/2017/08/inaccessible-youtube-embeds.html
